compiler/fir/fir2ir/src/org/jetbrains/kotlin/fir/backend/Fir2IrConverter.kt

```diff
@@ -138,6 +155,13 @@ class Fir2IrConverter(
             is FirProperty -> {
                 declarationStorage.createIrProperty(declaration, parent)
             }
+            is FirField -> {
+                if (declaration.origin == FirDeclarationOrigin.Synthetic) {
```

Comment:
This is repeated, and seems worth declaring as an extension property somewhere around here: https://github.com/JetBrains/kotlin/blob/master/compiler/fir/tree/src/org/jetbrains/kotlin/fir/declarations/FirDeclarationUtil.kt#L42
```
inline val FirDeclaration.isSynthetic: Boolean
    get() = origin == FirDeclarationOrigin.Synthetic
```

Reply:
done